frontend/src/components/Router/Router.tsx

```diff
@@ -55,7 +56,7 @@ export const Router: React.FC = () => {
         <LogPage />
       </Route>
       <Route path="/connections/:serviceID/users">
-        <UsersPage />
+        <UsersPageDevice />
```

Comment:
I believe this should be the service version.
```suggestion
        <UsersPageService />
```